src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -1591,8 +2031,10 @@ subroutine gsi_fv3ncdf_read(grd_ionouv,cstate_nouv,filenamein,fv3filenamegin)
           deallocate(uu2d_layout)
         enddo
       else
+       !write(6,*)' gsi_fv3ncdf_read_wang2: ',ilevtot,varname
         iret=nf90_inq_varid(gfile_loc,trim(adjustl(varname)),var_id)
         iret=nf90_get_var(gfile_loc,var_id,uu2d,start=startloc,count=countloc)
+       !write(6,*)' gsi_fv3ncdf_read_wang3: ',ilevtot,var_id,sum(uu2d)
```

Comment:
If this line won't be used in the future, please remove debug commented out line.

Reply:
Please remove debug write.